yummy_recipes/views.py

```diff
@@ -62,66 +92,209 @@ def login():
 def logout():
     ''' Log Out User '''
     session['logged_in'] = False
-    return redirect(url_for('home'))
+    session.pop('user')
```

Comment:
While `pop()` will do what you are trying to achieve, i.e., remove `user` from a list, you should use functions correctly. Here, `remove()` would have been more appropriate.

Reply:
`.remove()` wouldn't work since I'm not working with a list. Here I'm using [flask's session variables](http://flask.pocoo.org/docs/0.12/quickstart/#sessions)